src/common/datatypes/Set.cpp

```diff
@@ -11,6 +11,20 @@
 
 namespace nebula {
 
+Set Set::set_intersection(const Set& lhs, const Set& rhs) {
```

Comment:
Use camelcase naming.

Reply:
Consistent with std::set_intersection naming.

Reply:
Let's keep the consistence in the project